common/changes/@itwin/viewer-react/benp-optional-imodel-id_2022-09-06-18-35.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "Remove need for user-supplied iModelId",
+      "type": "patch"
```

Comment:
maybe make this a minor bump?
and change the comment so its says something like blank connection will be created if iTwinId is supplied and iModelId is not? might need to work on the wording there

Reply:
might want to update the other changelog entry as well

Reply:
Holding this open until I test changes in the test-app. Will add change then.

Reply:
Alright tests out fine. If you have time, take another pass at my responses/corrections and LGTM (let's get this merged :))

Reply:
lgtm!